maint/github_weekly_meeting.py

```diff
@@ -0,0 +1,123 @@
+#! /usr/bin/env python
+
+import sys, os
+from datetime import datetime, timedelta
+from pprint import pprint
+from getpass import getpass
+
+from github3 import login
+import github3
+
+
+def fetch(target_repo, last_num, gh):
+    orgname, reponame = target_repo.split('/', 1)
+
+    repo = gh.repository(orgname, reponame)
+    issues = repo.issues(state='all')
+
+    opened_issues = []
+    closed_issues = []
+
+    opened_prs = []
+    closed_prs = []
+
+    max_iss_num = 0
+    for issue in issues:
+        info = issue.as_dict()
+        iss_num = int(info['number'])
+        max_iss_num = max(max_iss_num, iss_num)
+
+        if iss_num <= last_num:
+            break
+        merged = False
+        if issue.pull_request_urls:
+            # Is PR?
+            merged = bool(info['pull_request'].get("merged_at"))
+            where = {'opened': opened_prs, 'closed': closed_prs}
+        else:
+            # Is Issues
+            where = {'opened': opened_issues, 'closed': closed_issues}
+
+        fmt = '- [{}#{}]({}) - {}'
```

Comment:
I would replaced this with an f-string and a `{' - merged' if merged else ''}` for readability.

Reply:
fixed in f681a847495e10b3e6f8e2eb71a2f233764ba298